packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -44,6 +44,16 @@ When you create an Okta expression, you can reference any property that exists i
 | `user.$property`                  | `user` reference to the Okta user<br>`property` the top-level property variable name<br>`id`, `status`, `created`, `lastUpdated`, `passwordChanged`, `lastLogin`   | `user.id`<br>`user.status`<br>`user.created`   |
 | `user.profile.$profile_property`  | `profile_property` reference to user profile property, including custom-defined properties  | `user.profile.firstName`<br>`user.profile.email`<br>           |
 
+### Okta Device Profile
+When you create an Okta expression, you can reference any property that exists in an Okta Device Profile, and EDR attributes.
+
+| Syntax                             | Definitions                                                                              | Examples                                                       |
+| --------                           | ----------                                                                               | ------------                                                   |
+| `device.profile.$profile_property`  | `profile_property` reference to device profile property  | `device.profile.managed`<br>`device.profile.registered`<br>           |
+| `device.provider.<vendor>.<signal>`| `device` reference to the Okta device<br>`vendor` the vendor such as `wtc` for Windows Security Center, `zta` for CrowdStrike| `device.provider.wsc.fireWall`<br>`device.provider.wsc.autoUpdateSettings`<br>`device.provider.zta.overall`   |
```

Comment:
```suggestion
| `device.provider.<vendor>.<signal>`| `vendor` references a vendor, such as `wtc` for Windows Security Center, or `zta` for CrowdStrike. <br>`signal` references the supported EDR signal by the vendor. | `device.provider.wsc.fireWall`<br>`device.provider.wsc.autoUpdateSettings`<br>`device.provider.zta.overall`   |
```

Reply:
I removed `device` reference due to the discrepancy in the example. Please add it back in with a better description, if required. Thanks!